docs/content/en/docs/examples/outbound-http.md

```
@@ -0,0 +1,32 @@
+
+```bash
+sudo ptcpdump -i any -c 5 'port 80 and host serverfault.com'
+```
+
+Run the capture and reproduce the traffic you care about. ptcpdump prints 
+the packets along with the responsible process (and container/pod if present), 
+revealing the binary and command line that triggered the outbound HTTP request.
+
+## Output Example
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)